clayui.com/src/components/IconSearch/index.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
We don't need this file really, we could just move everything from `IconSearch` over to this file.

Reply:
Alright, I was following the pattern in it's siblings, `LayoutNav`, `Sidebar`, etc.